### PR TITLE
refactor: to remove old key extractions config

### PR DESCRIPTION
**Pull request by yaswanth-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [X] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: crowdin/messages.json [hunk 1]
```diff
@@ -1833,7 +1833,6 @@
   "2057618836": "How to calculate stop loss and take profit level",
   "2057662499": "Profit with x500 multipliers",
   "2058893631": "Financial disclosure report",
-  "2060368992": "min read",
```
[the remark this answers is not in the document]
Comment (yaswanth-deriv) [reply crowdin/messages.json:1836]: I didn't remove this one. It was generated automatically once I created PR.

[the remark this answers is not in the document]
Comment (ali-hosseini-deriv) [reply crowdin/messages.json:1836]: just try to pull the latest master to make sure nothing has been changed

[the remark this answers is not in the document]
Comment (niloofar-deriv) [reply crowdin/messages.json:1836]: Seems like this is a extra extraction that's why it removed automatically

**Review by niloofar-deriv — APPROVED** [review]
LGTM :+1:

**Review by amir-deriv — APPROVED** [review]

File: crowdin/messages.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (yaswanth-deriv) [reply crowdin/messages.json]: I get it..When they use min read string _t_ is not using for localize..I added that now it is fine

**Review by mohsen-deriv — DISMISSED** [review]

**Pull request closed without merging**